data_storage/export_format.md

```diff
@@ -1,22 +1,28 @@
 Export Format
 -------------
 
-- **5.1.1** ![](/badge/req.png) A Toxcore data file saved in the root of the client's
-  working directory, with the file name `<profile_name>.tox`
+- **5.1.1** ![](/badge/req.png) The client will export files relative to a folder called
+  the export folder, which must be in a location accessible to the user.
 
-- **5.1.2** ![](/badge/req.png) If the client supports user avatars, they will be stored
-  under the `avatars/` directory in the root of the client's working directory.
+- **5.1.2** ![](/badge/rec.png) The user should be able to choose the location
+  of the export folder.
 
-- **5.1.3** ![](/badge/req.png) Avatars are named according to the following format:
+- **5.1.3** ![](/badge/req.png) A Toxcore data file saved in the root of the
+  export folder, with the file name `<profile_name>.tox`.
+
+- **5.1.4** ![](/badge/req.png) If the client supports user avatars, they will be stored
+  under the `avatars/` directory in the root of the export folder.
+
+- **5.1.5** ![](/badge/req.png) Avatars are named according to the following format:
   `PK.png`, where PK is the 64-character hexadecimal public key of its owner.
```

Comment:
Above, `<underscore_name>` was used to specify the variable part of a file name, now it's different. Choose one format.